packages/manager/apps/sign-up/src/routing.js

```diff
@@ -105,7 +105,7 @@ export const state = {
     trackError: /* @ngInject */ (atInternet, me) => (step, field) => {
       atInternet.trackEvent({
         event: 'ACCOUNT_CREATION_VALIDATION',
-        page: `accountcreation-${step}-error${field}-${me.legalform}`,
+        name: `accountcreation-${step}-error${field}-${me.legalform}`,
```

Comment:
I don't see name tag for trackEvent:

atInternet.trackEvent({
page: 'pageName', // page that raised the event
event: 'eventName', // raised event
level2: '1', // (optional) if not configured in defaults you must specify your project id
visitorId: '123', // (optional) set [identifiedVisitor.id](https://identifiedvisitor.id/) sent with each hit
});

Reply:
see on direct